documentation/concepts/statescripts.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: documentation
+menu: statescripts
+subtitle: State Scripts
+---
+# State Scripts
+{% include teams.html %}
+
+Over the lifetime of a project, many database objects may be created and destroyed across many migrations which leaves behind a lengthy history of migrations that need to be applied in order to bring a new environment up to speed.
+
+Instead, you might wish to add a single, cumulative script that represents the state of your database after all of those migrations have been applied without disrupting existing environments.
+
+[Flyway Teams Edition](/try-flyway-teams-edition) gives you a way to achieve this using **State Scripts**.
+
+## How it works
+
+Once you decide it's time to add such a cumulative script, instead of making it a regular [versioned migration](/documentation/concepts/migrations#versioned-migrations) it should be a state script, which is prefixed with `S` instead of `V`.
+
+With this prefix, these new scripts won't disrupt existing deployment pipelines and new environments will choose the latest state script as the starting point when you run `migrate`. Every migration with a version below the latest state script's version is marked as `below baseline`. <br/>
```

Comment:
I think get rid of 'these new scripts won't disrupt existing deployment pipelines and'

Reply:
Also the 'below baseline' thing might be confusing. Perhaps scripts below the state script should be 'IGNORED' instead.